deploy/README.md

```diff
@@ -85,10 +85,11 @@ The YAML files named below are to be found in the `deploy` folder of the Mayasto
 5.  Deploy the Mayastor and CSI components
     ```bash
     kubectl create -f nats-deployment.yaml
+    kubectl create -f mayastorpool.yaml
```

Comment:
not sure if it makes sense to add just one of them. There are two other yamls as well. Perhaps having just a note that CRD yamls are applied automatically by moac and a link to crds directory in moac would be sufficient. Whoever wants to apply them manually could do that then. But let's see what others think about it.

Reply:
My reasoning was that the pool instances are only thing which the user creates. The node and the volume instances are always created by moac.
Adding just the note also makes sense, whatever people think makes more sense / is clearer.

Reply:
I think there is potential to confuse the user at to what is being performed in this step.  To me, taking mayastorpool.yaml at face value suggests that it's going to do just that - establish a working pool instance(s), which is not the case.

I'm inclined to agree with Jan, that if we are going to have the user create *some* necessary CRDs explicitly as part of the installation step, rather than leave this to moac (and/or some other kind of operator or supervisory process), then we should define them *all* there and have a single "CRDs" yaml file which creates all such entities required.

Reply:
it doesn't need to be called mayastorpool right, it could be mayastorpoolcrd, common.yaml, or crd.yaml.
The tricky thing with combining them all is that we'd have to release that as an artifact perhaps, since they're all separate yaml files in the tree.

Reply:
Agreed.  Adding CRD to the filename would make sense.